index.html

```diff
@@ -0,0 +1,724 @@
+                </div>
+                <div id="part3_8"></div>
+            </div>
+            <div class="part4">
+                <ul>
+                    <li>Twitter shares fall 7% in pre-market trading after Trump's removal</li>
+                    <li>Winklevoss comes out of meditation to witness Bitcoin's record high</li>
+                    <li>CD Projekt Rekt: Cyberpunk 2077 tanks studio's share price nearly 50%</li>
+                    <li>CD Projekt stock down 20% since Cyberpunk reviews hit the internet</li>
+                    <li>Airbnb to be bigger than eBay after its long-awaited IPO</li>
+                </ul>
+            </div>
+        </div>
+        <div class="section4_items">
+            <div class="part1">
+                <a href="#">
+                    <h3>Distract ></h3>
+                </a>
+            </div>
+            <div id="part2_9">
+                <p>MOST POPULAR</p>
+                <a class="underline" href="#">Games to play on date night: Rule the galaxy together in Stellaris</a>
+                <p>Tristan Greene · 4 days ago</p>
+            </div>
+            <div class="part3">
+                <div class="before_image">
+                    <p>LATEST</p>
+                    <p><a href="#">The best vampire games and mods to play until Bloodlines 2 releases later this
+                            year</a></p>
+                </div>
+                <div id="part3_9"></div>
+            </div>
+            <div class="part4">
+                <ul>
+                    <li>Twitter shares fall 7% in pre-market trading after Trump's removal</li>
+                    <li>Winklevoss comes out of meditation to witness Bitcoin's record high</li>
+                    <li>CD Projekt Rekt: Cyberpunk 2077 tanks studio's share price nearly 50%</li>
+                    <li>CD Projekt stock down 20% since Cyberpunk reviews hit the internet</li>
+                    <li>Airbnb to be bigger than eBay after its long-awaited IPO</li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <div id="title3">
+        <div>
+            <a href="">
+                <h3>Latest deals ></h3>
+            </a>
+        </div>
+        <div>
+            <p><a href="">See all Deals</a></p>
+        </div>
+    </div>
+    <section id="section5">
+        <h2 class="none">hello</h2>
+        <div class="section5_items">
+            <img src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp2.stackassets.com/7a5a98dff30d5c38ecbc1700812eb2e1b93dce93/store/4b1f63289bd195a7bbafafbe36579fe597a5f1e7a0cb8165012908f3db74/sale_28088_primary_image.jpg"
+                alt="">
+            <div>
+                <p><a href="#"> The Ultimate 2021 White Hat Hacker Certification Bundle</a></p>
+                <span class="original_price">$1345</span>
+                <span class="new_price">$39.90</span>
+            </div>

```

Comment:
Each article in the Latest Deals section should have an image and a title (h1--h6) tag, both of them with a link (a tag). i.e.

```
h4>
  <a href="#" class="img-text">
    <p> The Ultimate 2021 White Hat Hacker Certification Bundle</p>
  </a>
</h4>
```

Reply:
Thanks for the suggestions. These changes have been made